### PR TITLE
Fix Thingy53 issues

### PR DESCRIPTION
**Pull request by czeslawmakarski**
This PR addresses several issues with Thingy:53.

### CODE REVIEW

**Review by jciupis — APPROVED** [review]

File: applications/machine_learning/configuration/nrf52840dk_nrf52840/prj_nus.conf [hunk 1]
```diff
@@ -82,7 +82,7 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
 CONFIG_ISR_STACK_SIZE=1536
 CONFIG_MAIN_STACK_SIZE=1024
 CONFIG_IDLE_STACK_SIZE=512
-CONFIG_BT_RX_STACK_SIZE=1600
+CONFIG_BT_RX_STACK_SIZE=2000
```
Comment (tejlmand) [inline applications/machine_learning/configuration/nrf52840dk_nrf52840/prj_nus.conf:85]: minor nit: This value is identical across platforms as far as I can tell.

So another option is to actually add:
```
config BT_RX_STACK_SIZE
        dafult 2000
```
to this file: https://github.com/nrfconnect/sdk-nrf/blob/main/applications/machine_learning/Kconfig

**Review by tejlmand — APPROVED** [review]
lgtm.

A minor nit, but not blocking for merge.

**Review by pdunaj — APPROVED** [review]

**Review by MarekPieta — APPROVED** [review]
Please make sure to also control PMIC PWM mode pin in a non-secure ML application

**Comment by jciupis** [comment]
@greg-fer documentation added.

**Review by greg-fer — APPROVED** [review]

**Comment by jfischer-no** [comment]
@czeslawmakarski Please update manifest

**Comment by jciupis** [comment]
@jfischer-no done.

**Comment by thst-nordic** [comment]
All Jenkins tests passed ( with exception for test-boot given by @ChristopherBittner )   
I'll force the merge.

**Pull request merged**